addons/website_mass_mailing/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_mailing_list_website_popup_ids(env):
+    popup = env['website.mass_mailing.popup']
+
+    sql = """
+    SELECT id, {popup_content} 
+    FROM mailing_list 
+    WHERE {popup_content} IS NOT NULL
+    """.format(popup_content=openupgrade.get_legacy_name('popup_content'))
+    env.cr.execute(sql)
+    rows = env.cr.fetchall()
+
+    for row in rows:
+        popup.create({
+            'mailing_list_id': row[0],
+            'popup_content': row[1]
+        })
```

Comment:
Also it better a create in batch:

```suggestion
    to_create = []
    for row in rows:
        to_create.append({
            'mailing_list_id': row[0],
            'popup_content': row[1]
        })
    if to_create:
        popup.create(to_create)
```